### ISSUE TITLE
cf.insert is not binary-safe

### ISSUE THREAD

**Issue by andrewschaaf** [simplereach/helenus#1]
```
cf.insert 'test-row', {'test-col3': new Buffer [0, 0xFF, 0x20]}, (e) ->
  cf.get 'test-row', (e, row) ->
    console.log row.get('test-col3')
```

&rarr;

```
{ name: <Buffer 74 65 73 74 2d 63 6f 6c 33>,
  value: <Buffer 00 fd 20>,
  timestamp: Thu, 15 Jan 1970 08:48:08 GMT,
  ttl: null }
```

With pycassa, I confirmed that the value saved is `00 FD 20`

**Comment by devdazed** [comment]
in your CF, what do you have the default validation type set as?

**Comment by devdazed** [comment]
This fix has been published in version 0.2.3
